doc/JSON-RPC-interface.md

```diff
@@ -0,0 +1,105 @@
+JSON-RPC Interface
```

Comment:
Please rename this file `json-rpc-interface.md` to make it consistent with the other files.

Reply:
I think matching `REST-interface.md` is the intention - should it be lowercase too?